test/e2e_node/gpu_device_plugin.go

```diff
@@ -65,19 +69,33 @@ var _ = framework.KubeDescribe("NVIDIA GPU Device Plugin [Feature:GPUDevicePlugi
 			if framework.NumberOfNVIDIAGPUs(getLocalNode(f)) < 2 {
 				Skip("Not enough GPUs to execute this test (at least two needed)")
 			}
+
+			By("Adding extended resource not managed by device plugin")
+			patch := []byte(fmt.Sprintf(`[{"op": "add", "path": "/status/capacity/%s", "value": "5"}]`, escapeForJSONPatch(nonDeviceResource)))
+			f.ClientSet.CoreV1().RESTClient().Patch(types.JSONPatchType).Resource("nodes").Name(framework.TestContext.NodeName).SubResource("status").Body(patch).Do()
+			Eventually(func() bool {
+				alloc, ok := getLocalNode(f).Status.Allocatable[nonDeviceResource]
```

Comment:
Is the node named `framework.TestContext.NodeName` the same as the node returned by `getLocalNode(f)`?

Reply:
For node_e2e, yes.